pkg/crc/machine/start.go

```diff
@@ -133,6 +133,38 @@ func growRootFileSystem(sshRunner *crcssh.Runner) error {
 
 	return nil
 }
+
+func configureSharedDirs(vm *virtualMachine, sshRunner *crcssh.Runner) error {
+	logging.Infof("configureSharedDirs")
+	sharedDirs, err := vm.Driver.GetSharedDirs()
+	if err != nil {
+		return err
+	}
+	for _, mount := range sharedDirs {
+		if !strings.HasPrefix(mount.Target, "/home") || !strings.HasPrefix(mount.Target, "/mnt") {
+			if _, _, err := sshRunner.RunPrivileged("Making / mutable", "chattr", "-i", "/"); err != nil {
+				return err
+			}
+		}
+		if _, _, err := sshRunner.RunPrivileged(fmt.Sprintf("Creating %s", mount.Target), "mkdir", "-p", mount.Target); err != nil {
+			return err
+		}
+		if !strings.HasPrefix(mount.Target, "/home") || !strings.HasPrefix(mount.Target, "/mnt") {
```

Comment:
yep, duplicated.

Reply:
This mimics the podman code https://github.com/containers/podman/blob/da1f47921685f40f1b26405278cbf9cb2d06fe09/pkg/machine/qemu/machine.go#L672-L679
I should at least add a similar comment to the one in the podman code. More importantly, the condition is wrong, it cannot be false (it's only false if mount.Target does have a /home prefix **and** a /mnt prefix which cannot happen).